vault/resource_gcp_auth_backend_role_test.go

```diff
@@ -156,24 +182,7 @@ func testGCPAuthBackendRoleCheck_attrs(backend, name string) resource.TestCheckF
 			return err
 		}
 
-		attrs := map[string]string{
```

Comment:
Do we need to pull this out? As you mentioned offline, this is the set of possible values so might be fine to keep it in here as a static set.

Reply:
Each test is sending the map of expected values now so I can remove the code that skips it.